auth_jwt.go

```diff
@@ -491,24 +503,6 @@ func (mw *GinJWTMiddleware) RefreshToken(c *gin.Context) (string, time.Time, err
 	return tokenString, expire, nil
 }
 
-// CheckIfTokenExpire check if token expire
-func (mw *GinJWTMiddleware) CheckIfTokenExpire(c *gin.Context) (jwt.MapClaims, error) {
```

Comment:
Why remove this func? See: https://github.com/appleboy/gin-jwt/pull/165

Reply:
Because there are two logic to check expiration.
* One for check MasRefresh 
* One for check Timeout expiration.
I think this function is not clear. So remove it.
And I didn't add 2 new functions. Because I didn't see other places to use the same logics. So I just put the logic there.

Reply:
Ok, I read the issue: https://github.com/appleboy/gin-jwt/pull/165.
I'm sure the best way to do. I think:
* gin-jwt should check timeout expiration in every request.

Maybe that's wrong? Em. It a complex situation. What is your idea?

Reply:
Reverted the function anyway.